src/components/EpicView/EpicView.tsx

```diff
@@ -1,27 +1,36 @@
-import { Group, Stack, Text, Title, ScrollArea, Box, Button, Center, Loader } from "@mantine/core";
+import { Group, Stack, Text, Title, ScrollArea, Box, Button, Center, Loader, TextInput } from "@mantine/core";
 import { useNavigate } from "react-router-dom";
-import { useState } from "react";
+import { ChangeEvent, useMemo, useState } from "react";
 import { useQuery } from "@tanstack/react-query";
+import { IconSearch } from "@tabler/icons-react";
 import { useCanvasStore } from "../../lib/Store";
 import { CreateIssueModal } from "../CreateIssue/CreateIssueModal";
 import { EpicWrapper } from "./EpicWrapper";
 import { getEpics } from "./helpers/queryFetchers";
 import { useColorScheme } from "../../common/color-scheme";
 import { RouteNames } from "../../route-names";
+import { filtersearch } from "./helpers/epicViewHelpers";
 
 export function EpicView() {
   const colorScheme = useColorScheme();
   const navigate = useNavigate();
   const projectName = useCanvasStore((state) => state.selectedProject?.name);
   const [createIssueModalOpened, setCreateIssueModalOpened] = useState(false);
   const projectKey = useCanvasStore((state) => state.selectedProject?.key);
+  const [search, setSearch] = useState("");
 
   const { isLoading: isLoadingEpics, data: epics } = useQuery({
     queryKey: ["epics", projectKey],
     queryFn: () => getEpics(projectKey),
     enabled: !!projectKey,
     initialData: [],
   });
+
+  const searchedEpicsWrapper = useMemo(
```

Comment:
```suggestion
  const searchedEpics = useMemo(
```
No need to call this a wrapper, since we are not using wrappers (unlike e.g. the backlog).

We probably do not need to use `useMemo` and can replace this just with a const definition. Give it a go!

Reply:
worked fine thanks!